adding-sessions.md

```diff
@@ -7,48 +7,121 @@
 In your everyday life at some point you've probably gotten an error page that said something like, "Session expired: log in again".
 
 **In a nutshell**, a session is two things:
-- An instance of you using a website, usually (but not always) with you having signed in somewhere.
+- An instance of you using a website, usually -- but not always -- with you having signed in somewhere.
 - A way for your Rails app to temporarily store some data about you "in memory" -- that is, not in Postgres -- and to associate the data with your particular computer. These data are called "session variables".
 
 Sessions work using cookies. We'll talk about that later.
 
 We can use them to make Rails "remember" you're logged in to Tunr.
 
-## 1. Create a Sessions controller
+## 1. Create A Sessions Controller
 
 When a User signs in, we'll call that a new "session". Signing in is very different from signing up: signing up changes the database, whereas signing in doesn't affect the database. Instead, it just compares the username and password typed by the user to the usernames and passwords in the database without changing them.
 
 Because it doesn't add anything to the database, we don't necessarily need a model for it.
 
-### [For now, just define the Session#new action](https://github.com/ga-wdi-exercises/tunr_rails_users/pull/3/files#diff-d5241d488259f32ecbe2f636133e5ddaR3)
+We will allow users to sign in by creating a form that corresponds to the `new` action in a Sessions controller. Let's begin by creating that controller and then the `new` action...
 
-This will be for the "Sign In" form.
+```bash
+$ rails g controller session
+```
+
+```ruby
+# app/controllers/sessions_controller.rb
 
-## [2. Create Sessions routes](https://github.com/ga-wdi-exercises/tunr_rails_users/pull/3/files#diff-21497849d8f00507c9c8dcaf6288b136R9)
+class SessionsController < ApplicationController
+  def new
+    @user = User.new
+  end
+end
+```
 
-This is one of relatively few cases in which we'll use `resource` (singular). The difference between `resource` and `resources` is that the routes have singular names, there's no `index` route, and there are no IDs in the `show`, `edit`, and `delete` routes.
+## 2. Create Sessions Routes
+
+Now let's reference sessions in `routes.rb`. We're going to do this a little differently than usual though...
+
+```rb
+resource :session   # Add this after the resources already in the file.
+```
+
+This is one of relatively few cases in which we'll use `resource` (singular). The difference between `resource` and `resources` is that the routes have singular names. That means there's no `index` route, and there are no IDs in the `show`, `edit` and `delete` routes.
 
 The implication is that a Resource is something that will not be saved to the database, and therefore does not need IDs.
```

Comment:
Ah, good to know! I'm going to reference that rake routes snippet in the lesson.